Mathlib/Data/ULift.lean

```diff
@@ -73,11 +74,20 @@ theorem «exists» {p : PLift α → Prop} : (∃ x, p x) ↔ ∃ x : α, p (PLi
   up_surjective.exists
 #align plift.exists PLift.exists
 
+lemma map_injective (hf : Injective f) : Injective (PLift.map f) :=
+  up_injective.comp $ hf.comp down_injective
+
+lemma map_surjective (hf : Surjective f) : Surjective (PLift.map f) :=
+  up_surjective.comp $ hf.comp down_surjective
+
+lemma map_bijective (hf : Bijective f) : Bijective (PLift.map f) :=
+  up_bijective.comp $ hf.comp down_bijective
```

Comment:
Please mention these in the PR description. Also, note `$` is deprecated in favor of `<|` in mathlib.